### PR TITLE
refactor(navigation): implemented left navigation new fabric styles

### PR DESCRIPTION
**Pull request by badejayayesubabu**
## Description

`<navigation>` : implement left navigation new fabric styles

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2172

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

File: src/scss/components/navigation/_index.scss [hunk 1]
```diff
@@ -56,17 +59,28 @@
...
     }
   }
 
   hx-reveal {
+    a.active {
+      background-color: $gray-200;
+      border-left: 0.125rem solid $purple-500;
+      color: $purple-500;
+      margin-left: 0.3rem;
+      padding-left: 3.25rem !important;
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:81]: We should avoid using `!important`...is it needed here?

File: src/scss/components/navigation/_index.scss [hunk 2]
```diff
@@ -56,17 +59,28 @@
...
     &:hover {
-      background-color: $gray-800;
-      color: $gray-0;
+      background-color: $gray-200;
+      color: $purple-500;
     }
   }
 
   hx-reveal {
+    a.active {
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:76]: You should be able to add styles based on the `:active` pseudo-class.  Try using `a:active` here.

File: docs/components/navigation/index.html [hunk 3]
```diff
@@ -45,7 +45,7 @@ <h2 id="left-navigation">Left (Vertical) Navigation</h2>
               <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
             </hx-disclosure>
             <hx-reveal id="section-1-2" open>
-              <a href="#">Link 3-1</a>
+              <a href="#" class="active">Link 3-1</a>
```
Comment (100stacks) [inline docs/components/navigation/index.html:48]: To show this state in our documentation, you can add documentation level styles.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/navigation/_index.scss (hunk 1 above)
Comment (badejayayesubabu) [reply src/scss/components/navigation/_index.scss]: Yes I removed , Please review

File: src/scss/components/navigation/_index.scss (hunk 2 above)
Comment (badejayayesubabu) [reply src/scss/components/navigation/_index.scss]: Changes have been done, Please review

File: docs/components/navigation/index.html (hunk 3 above)
Comment (badejayayesubabu) [reply docs/components/navigation/index.html]: Changes have been done, Please review

File: src/scss/components/navigation/_index.scss [hunk 4]
```diff
@@ -93,3 +99,28 @@
...
+
+#section-1 > a:active, #section-1 > a:focus {
+  margin-left: 1.25rem;
+  padding-left: 1.063rem;
+}
+
+#section-1-2 > a:active, #section-1-2 > a:focus {
+  margin-left: 2.5rem;
+  padding-left: 1.125rem;
+}
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:126]: @badejayayesubabu let's chat about this approach.  Normally, we avoid using `id`s.

Comment (badejayayesubabu) [reply src/scss/components/navigation/_index.scss]: Changes have been done. Please review

File: src/scss/components/navigation/_index.scss [hunk 5]
```diff
@@ -93,3 +99,32 @@
     }
   }//hx-reveal
 }
+
+// ===== SUB-LEFT NAVIGATION =====
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:103]: This is better and very close!

In the screenshot, please see:
* `Primary Selected` is bolder font when selected.
* `Secondary Selected`  has normal font when selected.
*  Please verify we have all nav states (e.g, disabled).
*  Please verify the proper padding when `selected`.
*  The specs include an example `w/ Icon`.  We should have styles for these.

Screen Shot 2021-04-28 at 11 42 16 PM

**Review by 100stacks — CHANGES_REQUESTED** [review]

**Pull request closed without merging**